modules/geographiclib/2.4.0.bcr.1/MODULE.bazel

```diff
@@ -0,0 +1,15 @@
+module(
+    name = "geographiclib",
+    version = "2.4.0.bcr.1",
+    bazel_compatibility = [">=7.2.1"],
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "cmake_configure_file", version = "0.1.0")
+bazel_dep(name = "rules_cc", version = "0.1.0")
+bazel_dep(name = "rules_python", version = "1.0.0")
+python = use_extension("@rules_python//python/extensions:python.bzl", "python")
```

Comment:
Would this make more sense to be in the `cmake_configure_file` module? geographic lib doesn't directly use python anywhere.

Reply:
@fmeum @kgreenek I've updated cmake configure file to use python3.12 and remove python dep in geographiclib.